extensions/amazon-aws/src/console.tsx

```diff
@@ -1,23 +1,39 @@
 import { ActionPanel, List } from "@raycast/api";
-import { readFile } from "fs/promises";
 import { useCachedPromise } from "@raycast/utils";
-import { AWS_URL_BASE } from "./constants";
+import { readFile } from "fs/promises";
 import { AwsAction } from "./components/common/action";
+import AWSProfileDropdown from "./components/searchbar/aws-profile-dropdown";
+import { AWS_URL_BASE } from "./constants";
 
 export default function Console() {
-  const { data: services, isLoading } = useCachedPromise(loadJSON);
+  const { data: services, isLoading, revalidate } = useCachedPromise(loadJSON);
 
   return (
-    <List isLoading={isLoading} searchBarPlaceholder="Filter services by name...">
+    <List
+      isLoading={isLoading}
+      searchBarPlaceholder="Filter services by name..."
+      searchBarAccessory={<AWSProfileDropdown onProfileSelected={revalidate} />}
+    >
       {services?.map((service) => (
         <List.Item
           key={service.uid}
           title={service.title}
           subtitle={service.subtitle}
           icon={service.icon.path}
+          keywords={service.match.split(" ")}
           actions={
             <ActionPanel>
-              <AwsAction.Console url={`${AWS_URL_BASE}${service.arg}`} />
+              <AwsAction.Console
+                url={
+                  typeof process.env.AWS_SSO_ACCOUNT_ID !== "undefined" &&
+                  typeof process.env.AWS_SSO_ROLE_NAME !== "undefined" &&
+                  typeof process.env.AWS_SSO_START_URL !== "undefined"
+                    ? normalizeUrl(
```

Comment:
I think we just need to just normalize the startUrl part. Also we can replace `console/?` with `console?`. The rest of the url is ok. So this exact string:

```typescript
`${normalizeUrl(process.env.AWS_SSO_START_URL)}console?account_id=${encodeURI(process.env.AWS_SSO_ACCOUNT_ID)}&role_name=${encodeURI(process.env.AWS_SSO_ROLE_NAME)}&destination=${encodeURI(AWS_URL_BASE + service.arg)}`
```

The previous URL after `:` does not need to be normalized.